server/command.go

```diff
@@ -262,12 +284,111 @@ func (p *Plugin) runHelpCommand(user *model.User) (string, error) {
 
 func (p *Plugin) runSettingCommand(args *model.CommandArgs, params []string, user *model.User) (string, error) {
 	if len(params) == 0 {
+		restrict, _, err := p.checkChannelPreference(args.ChannelId)
+		if err != nil {
+			p.API.LogError("Unable to check channel preference", "ChannelID", args.ChannelId, "Error", err.Error())
+			return "Error occurred while viewing zoom settings", nil
+		}
+
+		if restrict {
+			return "Updating zoom settings is disabled for this current channel.", nil
+		}
+
 		if err := p.sendUserSettingForm(user.Id, args.ChannelId, args.RootId); err != nil {
 			return "", err
```

Comment:
This command doesn't have to do with specific channels does it? It's more of a "here's your user settings" thing right? Wondering why these changes are needed